docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+services:
```

Comment:
Based on https://dev.to/hlappa/development-environment-for-elixir-phoenix-with-docker-and-docker-compose-2g17